README.md

```diff
@@ -50,6 +57,12 @@ Add the following to your POM file:
          <useTestNG>false</useTestNG>
          <!-- The naming scheme to use for the generated test classes.  One of 'simple' or 'feature-title' --> 
          <namingScheme>simple</namingScheme>
+         <!-- The class naming pattern to use.  Only required/used if naming scheme is 'pattern'.-->
+         <namingPattern>Parallel{c}IT</namingPattern>
+         <!-- Generate JUnitReRun runners instead of JUnit and TestNG single runner. --> 
+         <useJUnitReRun>true</useJUnitReRun>
+         <!-- It is meaningless to keep reruning without looking for root cause of the failure. Test case/features/scenarios won't pass magically so you can use max 5 rerun. -->
+			   <retryCount>2</retryCount>
```

Comment:
Yeah updated in Mojo class
  @Parameter(property = "retryCount",defaultValue= "0", required = true)